jetty-http/src/main/java/org/eclipse/jetty/http/HttpParser.java

```diff
@@ -1881,11 +1961,12 @@ public default void onComplianceViolation(HttpCompliance compliance, HttpComplia
     @SuppressWarnings("serial")
     private static class IllegalCharacterException extends BadMessageException
     {
-        private IllegalCharacterException(State state,byte ch,ByteBuffer buffer)
+        private IllegalCharacterException(State state,HttpTokens.Token token,ByteBuffer buffer)
         {
-            super(400,String.format("Illegal character 0x%X",ch));
             // Bug #460642 - don't reveal buffers to end user
```

Comment:
I think this line should be removed, it references a bugzilla ID that we don't use anymore, and there is `git blame` to figure out these things.